pyproject.toml

```diff
@@ -13,14 +13,14 @@ poetry-polylith-plugin = "poetry_polylith_plugin:PolylithPlugin"
 
 [tool.poetry.dependencies]
 python = "^3.7"
-poetry = {version = "^1.2.0a2", allow-prereleases = true}
+poetry = {version = "^1.2.0"}
 tomlkit = ">=0.7.0,<1.0.0"
 
 
 [tool.poetry.group.dev.dependencies]
-cachecontrol = "0.12.10"
-cleo = "1.0.0a4"
-flake8 = "^4.0.1"
+#cachecontrol = "0.12.10"
+#cleo = "1.0.0a5"
+#flake8 = "^4.0.1"
```

Comment:
Are these dependencies not needed anymore?

Reply:
My bad! I had some trouble with flake8. Being a formatter, I just dropped all dev deps for playing around.

This is the output I have. I also tried with the latest flake8, 5.0.4, but I got the same thing.

I guess this has to do with the `importlib-metadata>=1.1.0,<4.3;python_version<"3.8"` flake8 requirements that isn't considered for some reason on my side (I use python 3.10). [This issue](https://github.com/python-poetry/poetry/issues/5551) seems to point that no matter what is my python version, it's the python version set in the `pyproject.toml` that matter. When I bump the python version to `^3.8` it does work without issues.

```shell
Because no versions of flake8 match >4.0.1,<5.0.0
 and flake8 (4.0.1) depends on importlib-metadata (<4.3), flake8 (>=4.0.1,<5.0.0) requires importlib-metadata (<4.3).
And because poetry (1.2.0) depends on importlib-metadata (>=4.4,<5.0)
 and no versions of poetry match >1.2.0,<2.0.0, flake8 (>=4.0.1,<5.0.0) is incompatible with poetry (>=1.2.0,<2.0.0).
So, because poetry-polylith-plugin depends on both poetry (^1.2.0) and flake8 (^4.0.1), version solving failed.

```

Reply:
Aha, I see. There's probably a version mismatch with the new Poetry version.